cmd/controller/main.go

```diff
@@ -115,8 +115,9 @@ func main() {
 		"Burst size of events for the dynamic controller rate limiter.")
 
 	// reconciler parameters
-	flag.IntVar(&resyncPeriod, "dynamic-controller-default-resync-period", 36000,
-		"interval at which the controller will re list resources even with no changes, in seconds")
+	flag.IntVar(&resyncPeriod, "dynamic-controller-default-resync-period", 0,
```

Comment:
I would still keep this 10hours, just in case we miss events, or encounter watch errors.